components/matches/new/place-date/place-date.tsx

```diff
@@ -0,0 +1,52 @@
+import type { NextPage } from 'next';
+import {
+  Input, InputGroup, InputLeftElement,
+} from '@chakra-ui/react';
+import styles from './place-date.module.css';
+import { CalendarIcon, InfoOutlineIcon } from '@chakra-ui/icons';
+
+const PlaceDate: NextPage = () => {
+  return (
+    <div className={styles.form}>
+      <div>
+        <h3 className={styles.titleInput}>Place</h3>
```

Comment:
Use <Text> instead of h1:  https://chakra-ui.com/docs/components/input/usage#controlled-input

Reply:
Listo